sklearn/cluster/k_means_.py

```diff
@@ -885,13 +886,14 @@ def fit(self, X, y=None):
         Parameters
         ----------
         X : array-like or sparse matrix, shape=(n_samples, n_features)
-            Training instances to cluster.
+            Training instances to cluster. It must be noted that the data
+            will be converted to C ordering, which will cause a memory
+            copy if the given data is in fortran order.
```

Comment:
"copy if the given data is in fortran order" => "copy if the given data is not C-contiguous".

It could be Fortran-contiguous or no contiguous at all.

Reply:
Ah I missed this. Good point to remember.